### PR TITLE
added https://www.rituals.com/de-de/

### PR DESCRIPTION
**Pull request by paradonym**
added rituals (the online shop for kinda expensive wellbeing and bath stuff...) Special case for EU because of GDPR, I don't expect it to be the same way ouside of GDPR-areas.

### CODE REVIEW

**Comment by paradonym** [comment]
Maybe we should change the service name to "rituals Germany"

File: _data/sites.json [hunk 1]
```diff
@@ -8990,6 +8990,21 @@
...
+    {
+        "name": "Rituals.com",
+        "url": "https://service.rituals.com/s/contact?topic=a2L4L0000008xwDUAQ&language=de&country=DE",
+        "difficulty": "medium",
+        "notes": "You have to claim a GDPR case for the right to delete your data.",
+        "notes_tr": "verilerinizi silme hakkı için bir GDPR davası talep etmeniz gerekir.",
+        "notes_it": "devi rivendicare un caso GDPR per il diritto di cancellare i tuoi dati.",
+        "notes_pt_br": "você deve reivindicar um caso GDPR para o direito de excluir seus dados.",
+        "notes_es": "tiene que reclamar un caso GDPR por el derecho a eliminar sus datos.",
+        "notes_de" "Sie müssen einen DSGVO-Fall bzgl. des Rechts auf Löschung eigener Daten eröffnen.",
```
Comment (tupaschoal) [inline _data/sites.json:9002]:
```suggestion
        "notes_de": "Sie müssen einen DSGVO-Fall bzgl. des Rechts auf Löschung eigener Daten eröffnen.",
```

File: _data/sites.json [hunk 2]
```diff
@@ -8990,6 +8990,21 @@
         ]
     },
 
+    {
+        "name": "Rituals.com",
+        "url": "https://service.rituals.com/s/contact?topic=a2L4L0000008xwDUAQ&language=de&country=DE",
+        "difficulty": "medium",
+        "notes": "You have to claim a GDPR case for the right to delete your data.",
```
Comment (tupaschoal) [inline _data/sites.json:8997]: Does that mean that you do have to talk to a customer service person anyway?

Comment (paradonym) [reply _data/sites.json:8997]: You have to send a message via their contact form https://service.rituals.com/s/contact?topic=a2L4L0000008xwDUAQ&language=de&country=DE to file a GDPR Claim (as there's no button to delete an account anywhere). After the message a staff member from Rituals has to do the actual deletion inside their company administration tools. There's no verbal contact needed (thankfully).

Comment (tupaschoal) [reply _data/sites.json:8997]: I see, then I think its difficulty should be changed to hard.

**Review by tupaschoal — COMMENTED** [review]
Also note that we keep the entries sorted by key on the json, so you need to move yours up a little. Let me know if you need help.

**Comment by paradonym** [comment]
Close this because of pull request #920

**Pull request closed without merging**